modules/openapi-generator/src/main/resources/typescript-fetch/modelGeneric.mustache

```diff
@@ -157,4 +157,39 @@ export function {{classname}}ToJSON(value?: {{#hasReadOnly}}Omit<{{classname}},
     {{^hasVars}}
     return value;
     {{/hasVars}}
-}
+}{{#validationAttributes}}
+
+export const {{classname}}PropertyValidationAttributesMap: {
+    [property: string]: {
+        maxLength?: number,
+        minLength?: number,
+        pattern?: string,
+        maximum?: number,
+        exclusiveMaximum?: boolean,
+        minimum?: number,
+        exclusiveMinimum?: boolean,
+        multipleOf?: number,
+        maxItems?: number,
+        minItems?: number,
+        uniqueItems?: boolean
+    }
+} = { {{#vars}}{{#hasValidation}}
+    {{name}}: { {{#maxLength}}
```

Comment:
you can put the stuff on new lines, like

```
{{#maxLength}}maxLength: {{maxLength}},{{/maxLength}}
{{#minLength}}minLength: {{minLength}},{{/minLength}}
...
```

this makes the template nicer to read

Reply:
This generates additional empty lines for me. Is this intentional?
<details>
  <summary>Example</summary>

![Screenshot 2024-08-26 174154](https://github.com/user-attachments/assets/bc64d222-1537-4cbf-b9f9-e8b2acf95e8e)

</details>